controllers/cluster_controller.go

```diff
@@ -684,19 +684,44 @@ func GetPodSpec(cluster *fdbtypes.FoundationDBCluster, processClass fdbtypes.Pro
 	return internal.GetPodSpec(cluster, processClass, idNum)
 }
 
-func (r *FoundationDBClusterReconciler) hasFullReplication(cluster *fdbtypes.FoundationDBCluster) (bool, error) {
-	adminClient, err := r.DatabaseClientProvider.GetAdminClient(cluster, r)
+func hasDesiredFaultTolerance(adminClient AdminClient, cluster *fdbtypes.FoundationDBCluster) (bool, error) {
+	version, err := fdbtypes.ParseFdbVersion(cluster.Spec.Version)
 	if err != nil {
 		return false, err
 	}
-	defer adminClient.Close()
+
+	// TODO (johscheuer): https://github.com/FoundationDB/fdb-kubernetes-operator/issues/743
+	// For FDB versions lower than 6.2.0 we just approve the replacement.
+	// In 6.1 we only have machine information.
+	if !version.IsAtLeast(fdbtypes.FdbVersion{Major: 6, Minor: 2, Patch: 0}) {
```

Comment:
Rather than hardcoding a version here, can we add a method on the `FDBVersion` type? That's the general pattern we've followed, so it should make it easier to document the behavior of older versions in a centralized place, and remove it when we come to that issue.